src/autopas/LogicHandler.h

```diff
@@ -771,7 +773,15 @@ class LogicHandler {
    */
   std::vector<std::unique_ptr<std::mutex>> _bufferLocks;
 
+  /**
+   * Logger for configuration used and time spent breakdown of iteratePairwise.
+   */
   IterationLogger _iterationLogger;
+
+  /**
+   * Logger for FLOP count and hit rate.
+   */
+  FLOPLogger _FLOPLogger;
```

Comment:
```suggestion
  FLOPLogger _flopLogger;
```

Looks a bit confusing with the all-caps